### PR TITLE
Feature/user start lesson

### PR DESCRIPTION
**Pull request by drei001**
[Commands]
```
Create a virtual environment : virtualenv "e_learning"
Install requirements: "pip install requirements.txt"
Activate virtual environment: e_learning/Scripts/Activate
change directory to backend run "python manage.py runserver"
change directory to frontend run "npm run start"

--Important---
Please run this first to populate data
Generating fake data: "python manage.py createdata"
You can issue this command to generate fake categories ,questions and choices all in one. But please do note that you have to specify the true answer for each question, since we are just testing out if the client will sent the data back to the server, this will be optional 
```
[Pre-conditions]
### Login with your account do not use mine
[Expected Output]
### After picking you will be redirected to a lesson wizard form                  
image
### Finish all the questions and submit
image
### Data will be sent to the server. That's it!

### CODE REVIEW

File: frontend/src/redux/reducers/category.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  SUBMIT_ANSWER,
+} from "../actions/types";
+const category = (state = {}, action) => {
+  switch (action.type) {
+    case FETCH_CATEGORY_LIST:
+      return {
+        ...state,
+        ...action.payload,
+      };
+    case FETCH_CATEGORY:
```
Comment (inahsanjuan) [inline frontend/src/redux/reducers/category.js:13]: Create a separate reducer function for **"selected category"**, do not include in one function with the **"category list"**

File: frontend/src/components/fakedata.js [hunk 2]
```diff
@@ -1,23 +0,0 @@
...
-      first_name: firstName,
-      last_name: lastName,
-      avatar: avatar,
-    });
-  }
-
-  return { students: users };
-};
-
-export default generateUsers;
```
Comment (inahsanjuan) [inline frontend/src/components/fakedata.js:23]: I think this is helpful, why is this removed?

**Review by inahsanjuan — CHANGES_REQUESTED** [review]
Problems
- [ ] 1. After selecting a specific category, nothing is displayed. Upon checking code, I dont see any data generator for words, choice, etc

image

File: frontend/src/components/fakedata.js (hunk 2 above)
Comment (drei001) [reply frontend/src/components/fakedata.js:23]: This has been removed from the code, since I'm not using this anymore instead, I'm generating fake data on the backend.

Comment (inahsanjuan) [reply frontend/src/components/fakedata.js:23]: Ah I understand. I did not notice this is in frontend side. Lets leave it as is
